.github/workflows/build-and-deploy.yml

```diff
@@ -135,6 +135,8 @@ jobs:
           flavor: ${{ env.PACKAGE_TO_BUILD == 'mingw-w64-git' && 'build-installers' || 'full' }}
           architecture: ${{ env.ARCHITECTURE || 'x86_64' }}
           msys: ${{ env.REPO == 'MSYS2-packages' || env.PACKAGE_TO_BUILD == 'git-for-windows-keyring' }}
+          # We only have to clean up on self-hosted runners
+          cleanup: ${{ (runner.arch == 'ARM64' && true) || false }}
```

Comment:
I had a look at https://docs.github.com/en/actions/learn-github-actions/contexts#runner-context to see whether there would be _any_ indication of a self-hosted runner, but did not find any. Pity.

If only we could detect whether we're running on a self-hosted, _non-ephemeral_ runner, and set `cleanup: true` only then...

Would you mind losing the parentheses? We use the idiom 'A && B || C' (where `&&`/`||` follow [C++'s operator precedence order](https://en.cppreference.com/w/c/language/operator_precedence)) quite often, without parentheses, so that it looks inconsistent when we use them here.

Reply:
> Would you mind losing the parentheses? 

Done ✅ thanks for sharing the link to that documentation. I guess I got used to the [`no-mixed-operators`](https://eslint.org/docs/latest/rules/no-mixed-operators) rule in ESLint, which encourages using the parentheses to "clarfiy the developer’s intention" 😅

Reply:
Ah, that's where that comes from. Yes, I am an unfan of that rule ;-)